rerun_py/rerun_sdk/rerun/_log.py

```diff
@@ -86,9 +95,13 @@ def log(
     entity:
         Anything that can be converted into a rerun Archetype.
```

Comment:
Shouldn't this say: "anything that can be converted into a bundle of component batches" or something like that + maybe mention the `AsComponents` protocol?